lessons/module-2/closures.md

```diff
@@ -8,12 +8,19 @@ module: 2
 
 * Describe and speak to lexical scoping in JavaScript
 * Understand and implement closures
+* Identify scenarios where closures can be useful in code, and use closures to solve specific programming challenges
 
 ## Vocab
 
 - `lexical scope` also known as static scope
```

Comment:
This definition doesn't really help define lexical scope. I know we define it heavily later in the lesson, but it might be mice to beef this up a bit so students can easily find a full definition of the word when they reference this lesson later

Reply:
Bolstered the definition